resources/puppetlabs/lein-ezbake/template/global/ext/debian/ezbake.service.erb

```diff
@@ -44,13 +44,9 @@ UMask=027
 LimitNOFILE=<%= EZBake::Config[:open_file_limit] %>
 <% end -%>
 
-<% unless EZBake::Config[:debian][:pre_start_action].empty? -%>
-PermissionsStartOnly=true
```

Comment:
This would run the ExecStartPre command as root and could result into permission errors, so I removed it